src/sentry/models/project.py

```diff
@@ -257,16 +257,22 @@ def get_audit_log_data(self):
     def get_full_name(self):
         return self.slug
 
-    def get_notification_recipients(self, user_option):
+    def get_member_alert_settings(self, user_option):
+        """
+        Returns a list of users who have alert notifications explicitly
+        enabled/disabled.
+        :param user_option: alert option key, typically 'mail:alert'
+        :return: A dictionary in format {<user_id>: <int_alert_value>}
+        """
         from sentry.models import UserOption
-        alert_settings = dict(
-            (o.user_id, int(o.value))
-            for o in UserOption.objects.filter(
-                project=self,
-                key=user_option,
-            )
-        )
+        return {o.user_id: int(o.value) for o in UserOption.objects.filter(
+            project=self,
+            key=user_option,
+        )}
 
+    def get_notification_recipients(self, user_option):
+        from sentry.models import UserOption
```

Comment:
Should this import be in the function that relies on `UserOption`?

Reply:
It's used further down in the method as well

Reply:
I'm assuming we're avoiding a circular import there.